frappe/patches/v14_0/setup_likes_from_feedback.py

```diff
@@ -4,6 +4,9 @@
 def execute():
 	frappe.reload_doctype("Comment")
 
+	if not frappe.db.exists("Feedback"):
```

Comment:
```suggestion
	if not frappe.db.table_exists("Feedback"):
```

This isn't actually checking if a table exists.


![image](https://github.com/frappe/frappe/assets/9079960/c8d07a02-ebe9-4dae-8b65-3fdd8ab4ce3d)